docs/server/source/quickstart.md

```diff
@@ -6,6 +6,20 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
+Default database directory where mongodb stores its data: /data/db on Linux and OS X, \data\db on Windows.
```

Comment:
You can drop this line because we're assuming they're using Ubuntu 16.04 or similar (not Windows or Mac), as noted above. Some day, we might write detailed instructions for installation on many operating systems in another page. For now, we only support Ubuntu 16.04 or similar.